### DISCUSSION TITLE
What module format to use for extending features? CommonJS or ES6

### DISCUSSION THREAD

**Discussion by vaheeds** [hagopj13/node-express-boilerplate#90] [General]
I find this project very useful to extend my backend with it. Thanks to you developer. 
I am a c# developer and switch to node recently.  I see different approaches for using modules in JavaScript  out there that are confusing to me. I find that ES6 format is newer. Should I use this format or stick to the CommonJS format used in this project?

**Comment by hagopj13** [comment]
@vaheeds support for ES6 modules was added "recently" to Node.js. You can of course switch to using it. This should be a good guide: https://nodejs.medium.com/announcing-core-node-js-support-for-ecmascript-modules-c5d6dc29b663
